src/app/public/src/modules/nav/nav.component.ts

```diff
@@ -41,6 +41,11 @@ export class StacheNavComponent implements OnInit, StacheNav {
   public navigate(route: any): void {
     let extras: any = {};
 
+    if (route.isExternalLink) {
```

Comment:
Yeah I was on the fence about this anyway. I can just do the regex. I think that should cover us in most cases. I originally went with the boolean to account for any weirdness in URLs or different types of URLS. That could potentially be a mailto link or something, for example.